docs/adr/0014-apply-historical-events-to-aggregates.md

```diff
@@ -0,0 +1,40 @@
+# 14. Applying Historical Events to Aggregate Instances
+
+Date: 2020-10-19
+
+## Status
+
+Accepted
+
+## Context
+
+Event sourcing engines need to call `AggregateApplyEvent.ApplyEvent()` with
```

Comment:
I'm not sure this is correct `AggregateApplyEvent.ApplyEvent()`? Is there a `AggregateApplyEvent` type to call `ApplyEvent()` on?

Reply:
Fixed :+1: